Traceback (most recent call last):
[2022-02-26 09:02:03] ERROR:   File "C:\Users\Paulo\Downloads\Bot Bomb\bot\index.py", line 15, in <module>
    runMultiAccount()
[2022-02-26 09:02:03] ERROR:   File "C:\Users\Paulo\Downloads\Bot Bomb\bot\src\main_multi_account.py", line 56, in runMultiAccount
    "check_refresh_page": env.space['refresh_page'],
[2022-02-26 09:02:03] ERROR: KeyError: 'refresh_page'

